syft/pkg/cataloger/binary/default_classifiers.go

```diff
@@ -249,6 +249,14 @@ var defaultClassifiers = []classifier{
 		Package: "mariadb",
 		PURL:    mustPURL("pkg:generic/mariadb@version"),
 	},
+	{
+		Class:    "redmine-binary",
+		FileGlob: "**/version.rb",
```

Comment:
redmine ultimately is not a binary, so I don't think this belongs as a binary classifier -- is redmine a gem? ( I need to learn more about it), if so this feels like an enhancement to the ruby-base cataloger

Reply:
ahh I just saw the comments above on this -- I still think this shouldn't be detected by the binary cataloger

Reply:
Ok, fair enough